docs/install-github.md

```diff
@@ -35,6 +35,9 @@
    <details>
       <summary>Install Frogbot Using JFrog Pipelines</summary>
 
+   | Important: Using this method isn't recommended for open source projects. Learn more about it in the README under [Security note for pull requests scanning](../README.md#-security-note-for-pull-requests-scanning). |
```

Comment:
Important: Using this method isn't recommended for open source projects. Learn more about it in the README under [Security note for pull requests scanning](../README.md#-security-note-for-pull-requests-scanning
-->
Important: Using Frogbot with JFrog Pipelines isn't recommended for open source projects. Read more about it [here](../README.md#security-note-for-pull-requests-scanning).